.github/workflows/release.yml

```
@@ -32,7 +32,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v4.0.1
         with:
-          go-version: 1.21.x
+          go-version: 1.22.x
```

The `setup-go` action allows setting the Go version [directly from the `go.mod` file](https://github.com/actions/setup-go/tree/v4.0.1?tab=readme-ov-file#getting-go-version-from-the-gomod-file). It would avoid duplication and future desynchronization of the Go version:
```suggestion
          go-version-file: go.mod
```

---

That would be even better! @getsops/maintainers what do you think? Should we use `go.mod` as our source of truth?